src/painting/painter.rs

```diff
@@ -270,22 +269,22 @@ impl Painter {
 
         if use_ansi_coloring {
             self.stdout
-                .queue(SetForegroundColor(prompt.get_indicator_color()))?
-                .queue(SetAttribute(Attribute::Bold))?;
+                .queue(SetForegroundColor(prompt.get_indicator_color()))?;
         }
 
         self.stdout.queue(Print(&coerce_crlf(prompt_indicator)))?;
 
         if use_ansi_coloring {
             self.stdout
-                .queue(SetForegroundColor(prompt.get_prompt_right_color()))?
-                .queue(SetAttribute(Attribute::Bold))?;
+                .queue(SetForegroundColor(prompt.get_prompt_right_color()))?;
         }
 
         self.print_right_prompt(lines)?;
 
         if use_ansi_coloring {
-            self.stdout.queue(ResetColor)?;
+            self.stdout
+                .queue(SetAttribute(Attribute::Reset))?
+                .queue(ResetColor)?;
```

Comment:
Let's keep this reset, so a user provided prompt style doesn't leak into the main buffer.

Reply:
What do you mean? The reset is in there still, in fact I augmented it to reset the style attribute and reset the color. Are you waying you want to change this back to only the color reset?

Reply:
Ah my mental pattern matching was reading it wrong with all the `SetAttribute` deletions. My bad.

Thanks for adding this!